syntax = "proto3";

	message ClientRegistrationNotification
	{
	 uint32 typeNotification=1;
     string login=2;
	 string pass =3;
	 string userName=4;
	}

	message ClientSignInNotification
	{
	 uint32 typeNotification=1;
	 string login=2;
	 string pass =3;
	 uint64 sesionId=4;
	 uint64 userId=5;
	};

     message ClientSendMessageNotification
    {
	  uint32 typeNotification=1;
      uint64 senderId=2;
	  uint64 receiverId=3;
      string text=4;
    };

	message LogOutNotification
	{
	    uint32 typeNotification=1;
		string name=2;
                uint64 id=3;b
		uint64 sesionId=4;
	};

